src/parquet/types.h

```diff
@@ -130,6 +130,21 @@ struct SortOrder {
   enum type { SIGNED, UNSIGNED, UNKNOWN };
 };
 
+class ColumnOrder {
+ public:
```

Comment:
From where stems the need to have a full class for the column order? From looking at the code, it seems to me like using the enum should be sufficient.

Reply:
Yes, enum is sufficient for this PR. I added the full class for further extensions. The Java implementation did something similar: 
https://github.com/apache/parquet-mr/pull/435/files#diff-37183c50b2eaab3ca048f9c5954e5b52R26